apps/mull-ui/src/constants.ts

```diff
@@ -14,7 +14,7 @@ export const ROUTES = {
   MESSAGES: '/messages',
   GROUPCHAT: { url: '/messages/group-chat', displayName: 'Group Chat' },
   ANNOUNCEMENTS: { url: '/messages/announcements', displayName: 'Announcements' },
-  PROFILE: '/profile',
+  PROFILE: { profile: '/profile', editProfile: '/profile/edit' },
```

Comment:
💡
```suggestion
  PROFILE: { DISPLAY: '/profile', EDIT: '/profile/edit' },
```
 don't forget you can press `F2` to rename, and it should find all usages and change them for you